desktop/api/src/main/java/org/datacleaner/util/WidgetFactory.java

```diff
@@ -323,6 +333,41 @@ public static JXTextField createTextField(String promptText, int columns) {
         return tf;
     }
 
+    public static JXFormattedTextField createFormattedTextField(String promptText, final int columns, final Format format) {
+        JXFormattedTextField tf = new JXFormattedTextField(promptText);
+        // Stupid JXFormattedTextField will not pass along a formatter to the constructor.
```

Comment:
Maybe we should consider creating a separate class (DCFormattedTextField) for it then? Not a blocker for me, but maybe to consider if we touch the code anyway.